### PR TITLE
Fix Badly Broken Links Pt 1 

### PR DESCRIPTION
**Pull request by JRAlexander**
First batch of updated links from conversion (Redone to sync with Community PR)

## Summary

First batch of updated links from conversion (part 1 of badly converted links)

Note: this PR is Part 1 of several to make fixing  #2459 easier to review.

Migration broken links are identified by searching the repo for *qualifyHint*, which can be found [here](https://github.com/dotnet/docs/search?utf8=%E2%9C%93&q=qualifyhint&type=).

[Internal Review URL - File 1 (appcontextswitchoverrides-element)](https://review.docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element?view=netframework-4.7.1&branch=pr-en-us-3757)
[Internal Review URL - File 2  (table-valued-parameters)](https://review.docs.microsoft.com/en-us/dotnet/framework/data/adonet/sql/table-valued-parameters?branch=pr-en-us-3757)
[Internal Review URL - File 3 (linq-considerations-wcf-data-services)](https://review.docs.microsoft.com/en-us/dotnet/framework/data/wcf/linq-considerations-wcf-data-services?branch=pr-en-us-3757)
[Internal Review URL - File 4 (specifying-a-custom-crypto-algorithm)](https://review.docs.microsoft.com/en-us/dotnet/framework/wcf/extending/specifying-a-custom-crypto-algorithm?branch=pr-en-us-3757)
[Internal Review URL - File 5 (accessing-identity-information-inside-a-workflow-service)](https://review.docs.microsoft.com/en-us/dotnet/framework/wcf/feature-details/accessing-identity-information-inside-a-workflow-service?branch=pr-en-us-3757)
[Internal Review URL - File 6 (accessing-operationcontext-from-a-workflow-service)](https://review.docs.microsoft.com/en-us/dotnet/framework/wcf/feature-details/accessing-operationcontext-from-a-workflow-service?branch=pr-en-us-3757)
[Internal Review URL - File 7 (controlling-serialization-and-deserialization-with-serializationbinder)](https://review.docs.microsoft.com/en-us/dotnet/framework/wcf/feature-details/controlling-serialization-and-deserialization-with-serializationbinder?branch=pr-en-us-3757)
[Internal Review URL - File 8 (usage-of-the-switch-activity-with-custom-types)](https://review.docs.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/samples/usage-of-the-switch-activity-with-custom-types?branch=pr-en-us-3757)
[Internal Review URL - File 9 (microsoft.visualc.stlclr.generic.reversebidirectionaliterator-1)](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.visualc.stlclr.generic.reversebidirectionaliterator-1?view=netframework-4.7&branch=pr-en-us-3757)
[Internal Review URL - File 10 (microsoft.visualc.stlclr.generic.reverserandomaccessiterator-1)](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.visualc.stlclr.generic.reverserandomaccessiterator-1?view=netframework-4.7&branch=pr-en-us-3757)
[Internal Review URL - File 11 (system.activities.presentation.contextitemmanage)](https://review.docs.microsoft.com/en-us/dotnet/api/system.activities.presentation.contextitemmanager?view=netframework-4.7&branch=pr-en-us-3757)
[Internal Review URL - File 12 (system.data.sqltypes.sqlboolean)](https://review.docs.microsoft.com/en-us/dotnet/api/system.data.sqltypes.sqlboolean?view=netframework-4.7&branch=pr-en-us-3757)
[Internal Review URL - File 13 (system.data.sqltypes.sqlbyte)](https://review.docs.microsoft.com/en-us/dotnet/api/system.data.sqltypes.sqlbyte?view=netframework-4.7&branch=pr-en-us-3757)
[Internal Review URL - File 14 (system.data.sqltypes.sqlbytes)](https://review.docs.microsoft.com/en-us/dotnet/api/system.data.sqltypes.sqlbytes?view=netframework-4.7&branch=pr-en-us-3637)
[Internal Review URL - File 15 (system.data.sqltypes.sqlchars)](https://review.docs.microsoft.com/en-us/dotnet/api/system.data.sqltypes.sqlchars?view=netframework-4.7&branch=pr-en-us-3757)
[Internal Review URL - File 16 (system.data.sqltypes.sqldatetime)](https://review.docs.microsoft.com/en-us/dotnet/api/system.data.sqltypes.sqldatetime?view=netframework-4.7&branch=pr-en-us-3757)
[Internal Review URL - File 17 (system.data.sqltypes.sqlstring)](https://review.docs.microsoft.com/en-us/dotnet/api/system.data.sqltypes.sqlstring?view=netframework-4.7&branch=pr-en-us-3757)
[Internal Review URL - File 18 (system.device.location.geocoordinatewatcher)](https://review.docs.microsoft.com/en-us/dotnet/api/system.device.location.geocoordinatewatcher?view=netframework-4.7&branch=pr-en-us-3757)
[Internal Review URL - File 19 (system.identitymodel.tokens.sessionsecuritytokenhandler)](https://review.docs.microsoft.com/en-us/dotnet/api/system.identitymodel.tokens.sessionsecuritytokenhandler?view=netframework-4.7&branch=pr-en-us-3757)
[Internal Review URL - File 20 (system.printing.indexedproperties.printbooleanproperty)](https://review.docs.microsoft.com/en-us/dotnet/api/system.printing.indexedproperties.printbooleanproperty?view=netframework-4.7&branch=pr-en-us-3757)

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

File: xml/System.Data.SqlTypes/SqlBoolean.xml [hunk 1]
```diff
@@ -996,7 +996,7 @@
         <summary>Computes the bitwise AND operation of two specified <see cref="T:System.Data.SqlTypes.SqlBoolean" /> structures.</summary>
         <returns>The result of the logical AND operation.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:System.Data.SqlTypes.SqlBoolean.And%2A> oolean)?qualifyHint=True&autoUpgrade=False]]></format>
+          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <![CDATA[The equivalent method for this operator is <xref:System.Data.SqlTypes.SqlBoolean.And(System.Data.SqlTypes.SqlBoolean,System.Data.SqlTypes.SqlBoolean)?displayProperty=nameWithType>.]]></format>
```
Comment (mairaw) [inline xml/System.Data.SqlTypes/SqlBoolean.xml]: Repeated <![CDATA[The equivalent method for this operator is

File: xml/System.Device.Location/GeoCoordinateWatcher.xml [hunk 2]
```diff
@@ -201,7 +201,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Call <xref:System.Device.Location.GeoCoordinateWatcher.Dispose%2A> when you are finished using the <xref:System.Device.Location.GeoCoordinateWatcher> class?qualifyHint=False&autoUpgrade=True.  
+ Call <xref:System.Device.Location.GeoCoordinateWatcher.Dispose%2A> when you are finished using the <xref:System.Device.Location.GeoCoordinateWatcher> class.
```
Comment (mairaw) [inline xml/System.Device.Location/GeoCoordinateWatcher.xml]: For some reason this change isn't showing on stage but it looks good to me. @rpetrusha was having some issues with that too.
https://review.docs.microsoft.com/en-us/dotnet/api/system.device.location.geocoordinatewatcher.dispose?view=netframework-4.7.1&branch=pr-en-us-3757

**Review by mairaw — APPROVED** [review]
Looks wonderful! One thing to address and this can be merged.

File: xml/System.Device.Location/GeoCoordinateWatcher.xml (hunk 2 above)
Comment (JRAlexander) [reply xml/System.Device.Location/GeoCoordinateWatcher.xml]: Yeah, this was driving me nuts.

File: xml/System.Data.SqlTypes/SqlBoolean.xml (hunk 1 above)
Comment (JRAlexander) [reply xml/System.Data.SqlTypes/SqlBoolean.xml]: Great catch!

File: xml/System.Device.Location/GeoCoordinateWatcher.xml (hunk 2 above)
Comment (JRAlexander) [reply xml/System.Device.Location/GeoCoordinateWatcher.xml]: @mairaw - Thank you for the thoughtful comments and for the time spent reviewing! Much appreciated!!!

**Review by mairaw — APPROVED** [review]

**Pull request merged**